src/actions/MapActions.js

```diff
@@ -1,66 +1,122 @@
 import axios from 'axios';
+import { isMobile } from 'react-device-detect';
 import constants, { VERSION } from '../constants';
 import { getAuthHeader } from "../utils/Auth";
+import moment from 'moment';
 
 export const getMyMaps = () => {
     return async dispatch => {
-        const response = await axios.get(`${constants.ROOT_URL}/api/user/maps/`, getAuthHeader());
-        console.log("Got maps, response", response);
+        try {
+            const response = await axios.get(`${constants.ROOT_URL}/api/user/maps/`, getAuthHeader());
```

Comment:
postRequest is pretty nice, maybe we can have getRequest soon

Reply:
Yeah maybe this can be added when we address the connectivity issues more thoroughly